src/release_artifacts_resources/ios/cdk/cdk/credential_rotation_stack.py

```diff
@@ -1,29 +1,45 @@
-from aws_cdk import core as cdk
+from aws_cdk import core, aws_iam
 from cdk.credential_rotation.events_construct import EventsConstruct
 from cdk.credential_rotation.iam_construct import IAMConstruct
 from cdk.credential_rotation.lambda_construct import LambdaConstruct
 from cdk.credential_rotation.secretsmanager_construct import SecretsManagerConstruct
 from cdk.credential_rotation.cloudwatch_construct import CloudWatchConstruct
 
-
-class CredentialRotationStack(cdk.Stack):
+class CredentialRotationStack(core.Stack):
     def __init__(
         self,
-        scope: cdk.Construct,
+        scope: core.Construct,
```

Comment:
does this change require a new version of Python to run?

Reply:
I don't quite get that. This just changes the import name of the module. Why does it require a new python version?